src/en/PinkMuffinTL.lua

```diff
@@ -0,0 +1,87 @@
+-- {"id":4301,"ver":"1.0.8","libVer":"1.0.0","author":"MechTechnology"}
+
+local baseURL = "https://pinkmuffinyum.wordpress.com"
+
+local text = function(v)
+	return v:text()
+end
+
+local function shrinkURL(url)
+	return url:gsub("^.-pinkmuffinyum%.wordpress%.com", "")
+end
+
+local function expandURL(url)
+	return baseURL .. url
+end
+
+local function getPassage(chapterURL)
+	local doc = GETDocument(chapterURL):selectFirst("div.wp-site-blocks")
+	local chap = doc:selectFirst(".entry-content.wp-block-post-content")
+	local title = doc:selectFirst("h2.wp-block-post-title"):text()
+	chap:child(0):before("<h1>" .. title .. "</h1>")
+	return pageOfElem(chap, false, css)

```

Comment:
Where does css come from? Seems like it should be removed. Tend to think removing the style is the way to go, but have not checked whether it is needed.